docs/UserGuide.md

```diff
@@ -1,30 +1,38 @@
----
-layout: page
-title: User Guide
----
+# User Guide
 
-NUS Classes is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, NUS Classes can get your contact management tasks done faster than traditional GUI apps.
+NUS Classes is a desktop app for NUS Computing professors to manage their tasks and contacts. It includes task management features such as 
+adding and editing tasks, finding tasks by date or name, tagging tasks with appropriate tags and marking them as done. It also includes contact management features
+such as finding contacts, assigning contacts to specific tasks and tagging contacts. 
```

Comment:
```suggestion
creating tasks, tagging tasks, assigning contacts to tasks, and marking a tasks as complete or incomplete. It also includes contact management features
such as finding contacts, assigning contacts to specific tasks and tagging contacts. 
```

Since `filter` command is incomplete, should be left out.

Reply:
I think since we're shifting assigning contact to tasks up, line 5 shouldnt include assigning contacts to specific tasks anymore?